### PR TITLE
Build the navbar and header common components

### PR DESCRIPTION
**Pull request by muath-khatib**
-  some changes in filestructure for separate and distinguish files and creating folder assets for external files

Relates #4

### CODE REVIEW

**Review by Nizar7zak — APPROVED** [review]

**Comment by Nizar7zak** [comment]
Great work Muath <3

**Review by aaamra — APPROVED** [review]

**Review by reemsh2000 — APPROVED** [review]

**Review by emanNahed — APPROVED** [review]

File: client/src/components/common/NavTab/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const { Item, SubMenu } = Menu;
+
+const NAV = () => (
+  <Menu style={{ width: '34%' }} mode="horizontal">
+    <Item key="alipay">
+      <NavLink to="/home">Home</NavLink>
+    </Item>
+    <SubMenu key="shop" title="Shop" icon={<DownOutlined />}>
+      <Menu.ItemGroup>
+        <Menu.Item key="setting:1">shop 1</Menu.Item>
```
Comment (Mu7ammadAbed) [inline client/src/components/common/NavTab/Nav.jsx:19]: You should destructure `ItemGroup` and you already destructured `Item` 
```suggestion
      <ItemGroup>
        <Item key="setting:1">shop 1</Item>
```

File: client/src/components/common/NavTab/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <Item key="blog">
+      <NavLink to="/blog">Blog</NavLink>
+    </Item>
+    <Item key="contact-us">
+      <NavLink to="/contact-us">Contact us</NavLink>
+    </Item>
+  </Menu>
+);
+
+export default NAV;
```
Comment (Mu7ammadAbed) [inline client/src/components/common/NavTab/Nav.jsx:52]:
```suggestion
export default Nav;
```

File: client/src/components/common/NavTab/Logo.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import logo from '../../../assets/logo.png';
+
+import './Navbar.css';
+
+const Logo = () => (
+  <div className="header-navbar-logo">
+    <img className="header-navbar-logo-img" src={logo} alt="logo" />
```
[the remark this answers is not in the document]
Comment (muath-khatib) [reply client/src/components/common/NavTab/Logo.jsx:7]: The Image component in `antd` have an effect when user hover on the image and click to show the image in full size
and in my case the image it's static just a logo.

**Review by Mu7ammadAbed — APPROVED** [review]

**Pull request merged**